stlog/syslog.go

```diff
@@ -15,12 +16,14 @@ type kernelLogger struct {
 	level LogLevel
 }
 
+var errInitKlog = errors.New("init klog failed")
```

Comment:
There is no definite rule but usually errors like this should be exported. Especially for blackbox testing that would be mandatory but in our case it is optional. A linter should also not complain on exported package variable if it is an error.

Reply:
True. I guess this pkg needs further rework. E.g. putting the concrete logger implementations in own sub packages like stlog/klog or something. This klog thing is an implementation detail of the loggin package, and this particular error will not be populated to callers of stlog. This is why I left it unexported. Once it is in an own package, I would export it.